src/main/java/ua/javaPro/berezhnoy/hillelHomeWork6/HillelCollections.java

```diff
@@ -0,0 +1,75 @@
+package ua.javaPro.berezhnoy.hillelHomeWork6;
+
+import java.util.*;
+
+public class HillelCollections {
+    List<String> list = new ArrayList<>();
+
+    public static int countOccurance(List<String> arrList, String str) {
+        int result = Collections.frequency(arrList, str);
+        return result;
+    }
+
+    public static List<Integer> toList(Integer[] array) {
+        List<Integer> list = Arrays.asList(array);
+        return list;
+    }
+
+    public static List<Integer> findUnique(List<Integer> numbers) {
+        List<Integer> copyList = new ArrayList<>();
+        for (int i : numbers) {
+            if (!copyList.contains(i)) {
+                copyList.add(i);
+            }
+        }
+        return copyList;
+    }
+
+    public static List<String> calcOccurance(List<String> list) {
+        List<String> copyList = new ArrayList<>();
+        List<String> result = new ArrayList<>();
+
+        for (int i = 0; i < list.size(); i++) {
```

Comment:
```suggestion
        for (Integer value: list) {
```
немає сенсу заморочуватися із таким лупом (коли тобі не потрібен індекс)

Reply:
мне там нужен  индекс

Reply:
где именно? а `value` зачем тогда?

Reply:
![Новый точечный рисунок](https://user-images.githubusercontent.com/86191344/219720510-0f7ecb35-cea9-4168-9849-59f09be2746f.jpg)

Reply:
![Новый точечный рисунок](https://user-images.githubusercontent.com/86191344/219720757-2e59842e-c95c-4218-b991-ba466716d071.jpg)